config/routes.rb

```diff
@@ -38,6 +38,10 @@
   namespace :internal do
     get "/", to: redirect("/internal/articles")
 
+    authenticate :user, ->(user) { user.has_role?(:super_admin) } do
+      mount Blazer::Engine, at: "blazer"
```

Comment:
We may want a more friendly/memorable URL if we're not linking this from anywhere